lib/hal/led.c

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright and related rights waived via CC0
```

Comment:
nit: You'd normally still say who waves the right. See https://creativecommons.org/choose/zero/

The text can be trimmed but I assume you won't get around using your name?